biz.aQute.bndlib/src/aQute/bnd/help/Syntax.java

```diff
@@ -203,18 +220,24 @@
 			new Syntax(IMPORT_DIRECTIVE, "Experimental.", "", null, null)
 
 		),
+		new Syntax(TESTCASES, "The " + TESTCASES + " instruction is used to specify the list of test cases to run.",
+			TESTCASES + ": com.foo.bar.MyTest", null, Verifier.FQNPATTERN),
 		new Syntax(CONDITIONAL_PACKAGE, "The " + CONDITIONAL_PACKAGE
 			+ " works as private package but will only include the packages when they are imported. When this header is used, bnd will recursively add packages that match the patterns until there are no more additions.",
 			CONDITIONAL_PACKAGE + ": com.*", "${packages}", null),
 		new Syntax(CONDITIONALPACKAGE, "The " + CONDITIONALPACKAGE
 			+ " works as private package but will only include the packages when they are imported. When this header is used, bnd will recursively add packages that match the patterns until there are no more additions.",
 			CONDITIONALPACKAGE + ": com.*", "${packages}", null),
-		new Syntax(PRIVATE_PACKAGE, "The " + PRIVATE_PACKAGE
```

Comment:
This changes seems to have deleted `PRIVATE_PACKAGE` entirely which is incorrect.

Reply:
PRIVATE_PACKAGE has been duplicated mistakenly and that's why removed the redundant entry.

Reply:
If I look at the complete file from your commit, https://github.com/bndtools/bnd/blob/8cb860894495baabda7a0435062497e8a491b8e5/biz.aQute.bndlib/src/aQute/bnd/help/Syntax.java, I do not see `PRIVATE_PACKAGE` anywhere. Also, I only see `PRIVATE_PACKAGE` once in the current master.

Reply:
ah my bad. fixed.